demo/demo.ts

```diff
@@ -9,9 +9,9 @@
  */
 
 import * as fs from 'fs';
-import * as minimist from 'minimist';
+import { default as minimist } from 'minimist';
```

Comment:
`import minimist from 'minimist'`?

Reply:
Yeah, that's much better.  I think there was something before that kept me from doing that, but it seems to work.  Updated.